.gitignore

```diff
@@ -7,9 +7,9 @@ dune-workspace
 *.pyc
 *.cmly
 
-ocamlmerlin
-ocamlmerlin-server
-ocamlmerlin-lsp
+/ocamlmerlin
+/ocamlmerlin-server
+/ocamlmerlin-lsp
```

Comment:
?

Reply:
The current `.gitignore` ignores `ocamlerlin` in all subdirectories of the repositories. The leading `/` makes the .gitignore rule only apply to the root dir.

This is needed because I introduced an `ocamlmerlin` directory which would be ignored without this change.